test/index.js

```diff
@@ -52,12 +52,12 @@ function promise(fn, ...args) {
 
 function run(entry, limit, publicPath = "") {
   const plugin = url({limit, publicPath})
-  return rollup.rollup({
+  return rollup({
     entry,
     plugins: [plugin],
   }).then(bundle => bundle.write({
     dest,
-  })).then(() => plugin.write({dest}))
```

Comment:
Could you also make the same change in the README to reflect the simpler usage in the example?